docs/reference/troubleshooting/data/increase-node-capacity.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[increase-node-capacity]]
+== Not enough nodes to place all shard replicas
+
+Elasticsearch tries to keep copies of the data (index shard replicas) on different nodes so that
+if one node fails the data is still available on another node. If the index has a preferred tier,
```

Comment:
That (and more) will have been in the impact on the indicator that got them here, right? This part is more focused on just how to fix it isn't it?

Reply:
Oh sure, I was proposing we rephrase a bit this: 
> Elasticsearch tries to keep copies of the data (index shard replicas) on different nodes so that
if one node fails the data is still available on another node.

To something along the lines of:
>Elasticsearch tries to keep copies of the data (index shard replicas) on different nodes to protect against
hardware failure and increase the capacity to serve requests like searching for documents.